Example/src/LightboxExample.js

```diff
@@ -182,9 +204,15 @@ function ImageTransition({ activeImage, onClose }) {
   useEffect(() => {
     runOnUI(() => {
       'worklet';
-      animationProgress.value = withTiming(1, timingConfig, () => {
+
+      measureStuff();
+
+      // fixes flickering of the image
+      setTimeout(() => {
```

Comment:
that timeout looks ugly, wouldn't it be better to use something like `imageOpacity.value = withTiming(0, { duration: 16 });`?

Reply:
I'll try so, but not sure. It might be even some bug with ordering of updates because the fix is single frame delay.